src/Microsoft.AspNetCore.Diagnostics/project.json

```diff
@@ -7,17 +7,7 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true,
-    "compile": {
-      "include": "**/*.cs",
```

Comment:
Doing this causes the symbol packages to have the wrong structure (similar to https://github.com/dotnet/cli/issues/2829).